UserApi/UserApi/Helper/UserRole.cs

```diff
@@ -8,6 +8,7 @@ public enum UserRole
         [Display(Name = "Representative")] Representative = 2,
         [Display(Name = "Individual")] Individual = 3,
         [Display(Name = "Judge")] Judge = 4,
-        [Display(Name = "Case Admin")] CaseAdmin = 5
+        [Display(Name = "Case Admin")] CaseAdmin = 5,
+        [Display(Name = "Judicial Office Holder")] JudicialOfficeHolder = 5
```

Comment:
Should be 6 :)

Reply:
👍

Reply:
Whoops, figured out why my test was failing now haha